ui/base-tool/src/main/java/org/cru/godtools/base/tool/analytics/model/ShareActionEvent.kt

```diff
@@ -3,11 +3,8 @@ package org.cru.godtools.base.tool.analytics.model
 import org.cru.godtools.analytics.model.AnalyticsActionEvent
 import org.cru.godtools.analytics.model.AnalyticsSystem
 
-private const val ACTION_SHARE = "Share Icon Engaged"
-private const val ADOBE_SHARE_CONTENT = "cru.shareiconengaged"
+private const val ACTION_SHARE = "share_icon_engaged"
 
 object ShareActionEvent : AnalyticsActionEvent(action = ACTION_SHARE) {
-    override fun isForSystem(system: AnalyticsSystem) = system == AnalyticsSystem.ADOBE
-    override val adobeAttributes = mapOf<String, Any>(ADOBE_SHARE_CONTENT to 1)
```

Comment:
I'm assuming you are just dropping the `ADOBE_SHARE_CONTENT` param since it isn't needed?

Reply:
Yep, all the ```cru.``` params don't need to be carried over anymore.